src/NUnitTestAdapter/NUnit/NUnit3FrameworkDriver.cs

```diff
@@ -51,30 +48,33 @@ public class NUnit3FrameworkDriver : IFrameworkDriver
 
         AppDomain _testDomain;
         string _testAssemblyPath;
```

Comment:
Used to log the name. I commented out the logging in this copy. Fixing this line online.